lib/core/test/unit/Cardano/Wallet/DB/PureSpec.hs

```diff
@@ -9,7 +9,7 @@
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 
-module Cardano.Wallet.DB.MVarSpec
+module Cardano.Wallet.DB.PureSpec
```

Comment:
```suggestion
module Cardano.Wallet.DB.Pure.ImplementationSpec
```
… and corresponding change of filepath.

Looking at https://input-output-hk.github.io/adrestia/code/Coding-Standards#unit-test-files-names-match-their-corresponding-module , I interpret "within the same folder architecture" as a requirement to use the exact same module path before the last dot (`.`). 🤔